query/src/test/java/org/infinispan/query/indexing/IndexingOperationOffloadingTest.java

```diff
@@ -92,14 +92,13 @@ public void batchedPutAll() throws Exception {
          CompletableFuture<Void> execution = types.putAllAsync(chunk);
          log.info("Started: " + (c + 1) + " / " + CHUNKS_NUMBER + ". Elapsed: " + getElapsed(timeZero));
 
-         execution.whenComplete((unused, throwable) -> {
+         chunksExecutions[c] = execution.whenComplete((unused, throwable) -> {
```

Comment:
Does this test with the concurrency of `5` from the `SearchMappingHelper` or on the default 1000? It seems to issue 20 operations with 20 entries, which wouldn't fill the queue, I guess.

Reply:
It actually has a HS queue of 1 as configured in the `createCacheManager` method. Once the operation is submitted it will then be enqueued in our flatMap queue until it completes again.